src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/AcceptInvitation/AcceptInvitationCommandHandler.cs

```diff
@@ -77,6 +82,13 @@ public class AcceptInvitationCommandHandler : AsyncRequestHandler<AcceptInvitati
                 RelatedEntities = new List<Entity> { new Entity { Id =$"Account Id [{existing.AccountId}], User Id [{user.Id}]", Type = "Membership" } },
                 AffectedEntity = new Entity { Type = "Invitation", Id = message.Id.ToString() }
             });
+
+            await PublishUserJoinedMessage(existing.AccountId, $"{user.FirstName} {user.LastName}");
```

Comment:
i'm wondering if the user model needs a full name property like the memberView has as this could be a very common format that would be needed elsewhere.

Also should we be using the userId as two people could have the same name?

Reply:
Yep, done